tee-worker/ts-tests/integration-tests/common/di-utils.ts

```diff
@@ -94,9 +94,21 @@ export const createSignedTrustedCall = async (
     if (withWrappedBytes) {
         payload = u8aConcat(stringToU8a('<Bytes>'), payload, stringToU8a('</Bytes>'));
     }
-    const signature = parachainApi.createType('LitentryMultiSignature', {
-        [signer.type()]: u8aToHex(await signer.sign(payload)),
-    });
+
+    let signature;
+
+    // for bitcoin signature, we expect a hex-encoded `string` without `0x` prefix
+    // TODO: any better idiomatic way?
+    if (signer.type() === 'bitcoin') {
+        const payloadStr = u8aToHex(payload).substring(2);
+        signature = parachainApi.createType('LitentryMultiSignature', {
+            [signer.type()]: u8aToHex(await signer.sign(payloadStr)),
+        });
```

Comment:
so, if the u8a is signed, it will fail?

Reply:
No the important thing is unisat-wallet (and the underlying bitcore-lib) expects a `string` as signing message, unlike other signing mechanism where typically a u8a is used

Reply:
And our signing message is a `[u8; 32]`, so we need to convert it to string before signing